src/lib/validator/schemas.ts

```diff
@@ -25,4 +25,7 @@ export default {
     newPassword: PASSWORD,
     confirmPassword: `${PASSWORD}|same:newPassword`,
   },
+  validateCreateOrganisation: {
+    name: 'required',
```

Comment:
I think all the other fields are also required but I guess since we have ensured that the user inputs values before he can press the submit button then this is now redundant. 

Not sure if adding those other fields would aid readability in any way though. What do you think?

Reply:
Oh my bad, I forgot to revisit this. Yes, you are absolutely right, I agree with you as the validation will now be happening twice. How about I remove the validation happening here?